src/ExpressReceiver.ts

```diff
@@ -366,3 +413,41 @@ function parseRequestBody(stringBody: string, contentType: string | undefined):
 
   return JSON.parse(stringBody);
 }
+
+// Option keys for tls.createServer() and tls.createSecureContext(), exclusive of those for http.createServer()
+const httpsOptionKeys = [
```

Comment:
so only any one of these keys needs to be included to initiate the https server?

Reply:
yup, exactly.